### PR TITLE
fix(yt): Fix a few buttons

### PR DESCRIPTION
**Pull request by jopejoe1**
Fix theming of a few buttons

### CODE REVIEW

File: styles/youtube/catppuccin.user.css [hunk 1]
```diff
@@ -567,6 +567,15 @@
       border: none !important;
     }
 
+    /* Transcript Button */
+    .yt-spec-button-shape-next--call-to-action.yt-spec-button-shape-next--outline {
+      color: @accent-color !important;;
+      border-color: @text !important;;
```
Comment (isabelroses) [inline styles/youtube/catppuccin.user.css:573]:
```suggestion
      color: @accent-color !important;
      border-color: @text !important;
```

**Review by isabelroses — CHANGES_REQUESTED** [review]
Thanks for the fixes.

File: styles/youtube/catppuccin.user.css (hunk 1 above)
Comment (jopejoe1) [reply styles/youtube/catppuccin.user.css]: fixes

**Review by isabelroses — APPROVED** [review]

**Comment by isabelroses** [comment]
Thanks for the contribution.

**Pull request merged**